src/Simulation/TargetDefinitions/Intrinsic/ApplyControlledX.qs

```diff
@@ -31,6 +32,7 @@ namespace Microsoft.Quantum.Intrinsic {
     /// ```qsharp
     /// CNOT(control, target);
     /// ```
+    @TargetInstruction("cnot")
```

Comment:
This repo doesn't have any sample target package QIR output, so I can explain the effect of this change here. Without the `TargetInstruction` attribute, this callable would translate into a QIR declaration names something like `__quantum__qis____qsref1__applycontrolledx____body`. I say "named something like" because being `internal` means the name of the function automatically gets updated with `qsrefN` for some numeric value `N` in the Q# syntax tree to avoid collisions. With the `TargetInstruction` attribute defined, the provided string "cnot" overrides the name-mangled declaration and instead the QIR will have `__quantum__qis__cnot` which is both more simply named and deterministic such that implementing the definition of the declaration is possible.